interface/locales/de/common.json

```
@@ -217,47 +217,47 @@
   "extensions": "Erweiterungen",
   "extensions_description": "Installiere Erweiterungen, um die Funktionalität dieses Clients zu erweitern.",
   "fahrenheit": "Fahrenheit",
-  "failed_to_add_location": "Failed to add location",
+  "failed_to_add_location": "Fehler beim Hinzufügen der Location",
   "failed_to_cancel_job": "Aufgabe konnte nicht abgebrochen werden.",
   "failed_to_clear_all_jobs": "Alle Aufgaben konnten nicht gelöscht werden.",
   "failed_to_copy_file": "Fehler beim Kopieren der Datei",
   "failed_to_copy_file_path": "Dateipfad konnte nicht kopiert werden",
   "failed_to_cut_file": "Fehler beim Ausschneiden der Datei",
-  "failed_to_delete_rule": "Failed to delete rule",
+  "failed_to_delete_rule": "Fehler beim Löschen der Regel",
   "failed_to_download_update": "Update konnte nicht heruntergeladen werden",
   "failed_to_duplicate_file": "Datei konnte nicht dupliziert werden",
   "failed_to_generate_checksum": "Prüfsumme konnte nicht generiert werden",
   "failed_to_generate_labels": "Labels konnten nicht generiert werden",
   "failed_to_generate_thumbnails": "Vorschaubilder konnten nicht generiert werden",
   "failed_to_load_tags": "Tags konnten nicht geladen werden",
-  "failed_to_open_file_title": "Failed to open file",
-  "failed_to_open_file_body": "Couldn't open file, due to an error: {{error}}",
-  "failed_to_open_file_with": "Failed to open file, with: {{data}}",
+  "failed_to_open_file_title": "Fehler beim Öffnen der Datei",
+  "failed_to_open_file_body": "Konnte die Datei aufgrund des Fehlers nicht Öffnen: {{error}}",
```

```suggestion
  "failed_to_open_file_body": "Konnte die Datei aufgrund des Fehlers nicht öffnen: {{error}}",
```

What's up with these random capitalisations all over the place